utilities/input_tools.py

```diff
@@ -4,6 +4,7 @@
 from utilities import boostHistHelpers as hh,logging
 import numpy as np
 import os
+import hdf5plugin
```

Comment:
It's needed to load the compression plugins for hdf5.  (In the next singularity update it could be that this is no longer required since we can handle this at the level of environment variables.)

Reply:
it's a bit nasty since it's needed but not "used." I think it's imported in nerf so it only would fail if you ran this independently